tests/integration/targets/ios_facts/tests/cli/all_facts.yaml

```diff
@@ -6,7 +6,7 @@
   cisco.ios.ios_facts:
     gather_subset:
       - all
-    provider: '{{ cli }}'
```

Comment:
why is this file updated, as I see there's no change here wrt to the module leaving the addition of module name under `set_fact`, if not intended can u revert the change.

Reply:
That is the tox scan was reformatting it, let me revert the changes.

Reply:
tox is updating the formatting, and the files changed produces warnings or gets reformatted on tox scan. 
As it is not making any of the tests fail. Gate'ing it.